hydrosdk/application.py

```diff
@@ -18,7 +20,7 @@ def streaming_params(in_topic, out_topic):
     }
 
 
-class Application:
+class Application(Predictable):
```

Comment:
❌ Applications do not support shadowless predictions.

Reply:
Fixed

Reply:
So, Applications still have `predictor` method with `monitorable` param, and user can create an invalid predictor this way?

Reply:
Fixed